### PR TITLE
chore(dragdrop): allow further styling

### PR DESCRIPTION
**Pull request by jinlee93**
### Summary:
- [slack discussion](https://czi-edu.slack.com/archives/CTFV79JH4/p1663958436719399?thread_ts=1663181721.585879&cid=CTFV79JH4)
- This allows classnames to be passed in to style the dragdrop containers, containers wrapper, dragdrop containers columns, and does a little cleanup regarding

https://user-images.githubusercontent.com/86632227/192334637-18577425-98ff-4c12-8eb1-af3447d7348d.mov

- DragDrop itself is not the easiest to read and could definitely [use some clean up or even a refactor](https://app.shortcut.com/czi-edu/story/216547)
### Test Plan:
- Adds interactive story in DragDrop showing the customizable wrapper, container, and column
- No other visual regression
- Unit tests pass

### CODE REVIEW

File: src/components/DragDropContainer/DragDropContainer.tsx [hunk 1]
```diff
@@ -8,16 +8,6 @@ import DragDropContainerHeader from '../DragDropContainerHeader';
...
-  /**
-   * CSS class names that can be appended to the component.
-   */
-  className?: string;
-  /**
-   * Column class names that can be appended to the component.
-   *
-   * Used to add additional styles to the column.
-   */
-  columnClassName?: string[];
```
Comment (jinlee93) [inline src/components/DragDropContainer/DragDropContainer.tsx:20]: These props were never used, extrapolated from the `container` object

File: src/components/DragDrop/DragDropTypes.ts [hunk 2]
```diff
@@ -17,9 +17,10 @@ export interface Containers {
 }
 
 export interface ContainerType {
+  className?: string;
+  columnClassName?: string;
+  emptyContent?: React.ReactNode;
+  header?: React.ReactNode;
```
Comment (jinlee93) [inline src/components/DragDrop/DragDropTypes.ts:23]: Having types in this separate file and no comments is one result of not using `children`, an example of a need for refactor

File: src/components/DragDrop/DragDrop.tsx [hunk 3]
```diff
@@ -298,7 +305,7 @@ export const DragDrop = ({
               ref={provided.innerRef}
             >
               <div
-                className={styles['drag-drop__inner']}
+                className={innerClassName}
```
Comment (jinlee93) [inline src/components/DragDrop/DragDrop.tsx:308]: Another example for need to refactor is this extraneous `div` wrapper, could probably have functionality on parent element directly (difficult to move in this scope)

**Comment by dierat** [comment]
Is the video in this PR's summary the right one? It looks like it's about the `Drawer` component opening and closing.

File: src/components/DragDrop/DragDrop.module.css [hunk 4]
```diff
@@ -60,7 +60,7 @@
 /**
  * Outer container for DragDrop component; provides <DragDropContext />, which wraps droppable components (i.e. containers) and provides them with context.
  */
-.drag-drop__inner {
```
Comment (dierat) [inline src/components/DragDrop/DragDrop.module.css:63]: Can we get a comment about this `:where` usage?

File: src/components/DragDrop/DragDrop.stories.tsx [hunk 5]
```diff
@@ -490,3 +493,206 @@ export const HoveredHandle: StoryObj<Args> = {
     unstyledItems: true,
   },
 };
+
+const InteractiveDragDrop = () => {
```
Comment (dierat) [inline src/components/DragDrop/DragDrop.stories.tsx:497]: Thanks for adding this story; it's super helpful for testing this PR.

**Review by dierat — APPROVED** [review]
>DragDrop itself is not the easiest to read

Agreed 😬 I'm not fully grasping how this component is organized, but the interactive story works great and the changes look reasonable 👍 Thanks for picking up this request ❤️

**Pull request merged**